File: 240131/평균점수 구하기/print-average.c
#include <stdio.h>

int main() {
    // 실수 배열 선언
    double arr[8];

    // 변수 선언
    double sum = 0;

	// 8개의 실수를 배열에 입력받아 저장합니다.
    for(int i = 0; i < 8; i++){
        scanf("%lf", &arr[i]);
    }
	
	// 배열에 저장된 원소들의 합을 구합니다.
	for(int i = 0; i < 8; i++){
        sum += arr[i];
    }

    // 평균을 구하여 출력합니다.
    printf("%.1lf", sum / 8);

    return 0;
}

File: 241107/평균점수 구하기/print-average.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // 실수 배열 선언
        double[] arr = new double[8];

        // 변수 선언
        double sum = 0;

        // 8개의 실수를 배열에 입력받아 저장합니다.
        for(int i = 0; i < 8; i++) {
            arr[i] = sc.nextDouble();
        }
        
        // 배열에 저장된 원소들의 합을 구합니다.
        for(int i = 0; i < 8; i++){
            sum += arr[i];
        }
        
        // sum을 출력합니다.
        System.out.printf("%.1f", sum / 8);
    }
}